### PR TITLE
Update Alpha Company Loadouts + PREAPPROVED BY CoC

### PR DESCRIPTION
**Pull request by Robot-Panda15**
**When merged this pull request will:**
Overhaul and update ACo Loadouts >List acquired from BTN CO

### CODE REVIEW

File: cScripts/Loadouts/CfgLoadouts_Alpha.hpp [hunk 1]
```diff
@@ -123,311 +122,397 @@ class Cav_B_A_Officer_F  : Cav_B_Alpha_base_F {
...
+    };
+ 
     gps[] = {"ItemcTab"};
+ 
 };
 class Cav_B_A_Helicopter_Tra_coPilot_F : Cav_B_Alpha_base_F {
     backpack[] = {"B_Kitbag_mcamo"};
-
+    headgear[] = {"rhsusf_hgu56p_visor_green"};
+    goggles[] = {"rhsusf_shemagh_tan"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Alpha.hpp]: Move to inventory to allow player custom (`items` config array)

File: cScripts/Loadouts/CfgLoadouts_Alpha.hpp [hunk 2]
```diff
@@ -123,311 +122,397 @@ class Cav_B_A_Officer_F  : Cav_B_Alpha_base_F {
...
+ 
     preLoadout = "[(_this select 0), 'officer'] call cScripts_fnc_setPreInitPlayerSettings;";
 };
-
+ 
 class Cav_B_A_Helicopter_Tra_Pilot_F : Cav_B_Alpha_base_F {
     backpack[] = {"B_Kitbag_mcamo"};
-    nvgs[] = {"rhsusf_ANPVS_15"};
+    headgear[] = {"rhsusf_hgu56p_visor_green"};
+    goggles[] = {"rhsusf_shemagh_tan"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Alpha.hpp]: Move to inventory to allow player custom (`items` config array)

File: cScripts/Loadouts/CfgLoadouts_Alpha.hpp [hunk 3]
```diff
@@ -123,311 +122,397 @@ class Cav_B_A_Officer_F  : Cav_B_Alpha_base_F {
...
     };
-    nvgs[] = {"rhsusf_ANPVS_15"};
+ 
     gps[] = {"ItemAndroid"};
 };
 class Cav_B_A_Helicopter_Tra_CrewChief_F : Cav_B_Alpha_base_F {
-    headgear[] = {"rhsusf_hgu56p_visor_mask_black"};
     backpack[] = {"B_Carryall_cbr"};
+    headgear[] = {"rhsusf_hgu56p_visor_mask_Empire_black"};
+    goggles[] = {"rhsusf_shemagh_tan"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Alpha.hpp]: Move to inventory to allow player custom (`items` config array)

File: cScripts/Loadouts/CfgLoadouts_Alpha.hpp [hunk 4]
```diff
@@ -123,311 +122,397 @@ class Cav_B_A_Officer_F  : Cav_B_Alpha_base_F {
...
-
+ 
     preLoadout = "[(_this select 0), 'alpha', 0, 1] call cScripts_fnc_setPreInitPlayerSettings;";
 };
 class Cav_B_A_Helicopter_Tra_DoorGunner_F : Cav_B_Alpha_base_F {
-    headgear[] = {"rhsusf_hgu56p_visor_mask_black"};
     backpack[] = {"B_Kitbag_mcamo"};
-
+    headgear[] = {"rhsusf_hgu56p_visor_mask_black"};
+    goggles[] = {"rhsusf_shemagh_tan"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Alpha.hpp]: Move to inventory to allow player custom (`items` config array)

**Comment by ColdEvul** [comment]
Reason behind the removal of NVG and headgere? Just a security question :)

Have you tested this ingame? If so I'll merge this :)

**Comment by Robot-Panda15** [comment]
Haven't tested yet in game, been busy with other focused.

**Comment by Robot-Panda15** [comment]
Also I removed the NVGs and head gear and added them into the player inventory instead

**Comment by ColdEvul** [comment]
headgear is fine to be on there head cause now they get dubble due to the base class it's just eyewear that have its own system and special behavior :)

**Comment by Robot-Panda15** [comment]
Tested, fixed, ready to go.

**Pull request merged**